### DISCUSSION TITLE
Assets 中的 UI 图片是手动处理的吗？

### DISCUSSION THREAD

**Discussion by perhydrol** [LmeSzinc/AzurLaneAutoScript#2171] [Q&A] [answered]
如图中的各种 UI 图片，都是截图后手动打上黑色遮罩的吗？我暂时没有发现生成相关图片的代码。
![](https://s3.bmp.ovh/imgs/2023/01/25/f4d53cd5c6563a55.png)
C:\USERS\K\DOCUMENTS\PROJECT\AZURLANEAUTOSCRIPT-MASTER\ASSETS
├─cn
│  ├─campaign
│  ├─combat
│  ├─commission

**Comment by bookbug666** [comment, marked as answer]
https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/4.1.-Detection-objects
參考這個，修改檔案運行button_extract寫入各路徑的assets.py

**Reply by perhydrol** [reply]
噢，感谢回答。 我没仔细看 WIKI (x_x)
